### ISSUE TITLE
Audit #1 – General Findings – screen reader and keyboard users get no hekp on SPA route change

### ISSUE THREAD

**Issue by marcus-herrmann** [owncloud/web#4409]
Connected to Audit 1, see #4300 
2 General findings, URL: https://ocis-a11y.owncloud.works/#/files/list

# Issue
See https://github.com/owncloud/phoenix/issues/4300#issuecomment-729753253 Update Dec 2nd, 2020: That no focus management happens after route change could possibly be a violation of BITV 2.4.3a, Sensible Focus Order.

# Remediation
At least change the focus after successful route change: https://marcus.io/blog/accessible-routing-vuejs
I recommend an even more holistic approach for keyboard users as well: https://marcus.io/blog/improved-accessible-routing-vuejs

**Comment by pascalwengerter** [comment]
Closing this since #5018 is merged and we're waiting for feedback (and will open new tickets with the requested changes from the `a11y` audit)
